starter-code/app.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+
+const app = express();
+
+app.use(express.static('public'));
+
+
+app.get('/', (request, response, next) => {
+    response.sendFile(__dirname + '/views/home-page.html');
+  });
+  
+  
+app.get('/about', (request, response, next) => {
+  response.sendFile(__dirname + '/views/about-page.html');
+});
+
+
+app.get('/photos', (request, response, next) => {
+  response.sendFile(__dirname + '/views/photo-gallery.html');
+});
+
+app.listen(3000, () => {
+    console.log('server is up and running!');
+  });
+
+//   button.onclick = function(){console.log ("that's enough internet for today")};
```

Comment:
if you wanted to add extra JS for doing things, then you'd need to create another folder within the public file called javascript and then create a file within that folder.  
for example: 

public (folder) => javascript (folder) => script.js (file)

and then to link that file in HTML you would need to access it like you do for photos 

``"/javascript/script.js" ``